src/SimEcu.cpp

```diff
@@ -0,0 +1,41 @@
+#include "SimEcu.h"
+
+SimEcu::SimEcu() {}
+
+void SimEcu::begin(){
```

Comment:
Would it be possible to pass a pointer of a `Serial` type here for devices with multiple serial ports (or even `SoftwareSerial`)? It'd make this a lot more flexible.

Reply:
Done!